src/functions/pollJournals/framework/repo/mysql/test-slot-repository.ts

```diff
@@ -1,188 +1,178 @@
+         ts.extended_ind as special_needs_extended_test, ts.special_needs_code as special_needs_code,
+         ari.booking_seq as booking_seq,
+         ari.check_digit as check_digit, v.cab_seat_count, v.passenger_seat_count,
+         v.height_m as height_metres, v.length_m as length_metres, v.width_m as width_metres,
+         v.gearbox_code,
+         ts.test_category_ref as vehicle_category, i.individual_id as candidate_id,
+         title_ref.item_desc1 as candidate_title, i.first_forename as candidate_first_name,
+         i.second_forename as candidate_second_name, i.third_forename as candidate_third_name,
+         i.family_name as candidate_surname, i.driver_number as candidate_driver_number,
+         ccd.contact_details_id as candidate_cd_id, ccd.prim_tel_voicemail_ind, ccd.primary_tel_number,
+         ccd.sec_tel_voicemail_ind, ccd.secondary_tel_number, ccd.mobile_voicemail_ind, ccd.mobile_tel_number,
+         ccd.email_address as cand_email, cand_addr.address_id as candidate_addr_id, cand_addr.address_line_1,
+         cand_addr.address_line_2, cand_addr.address_line_3, cand_addr.address_line_4, cand_addr.address_line_5,
+         cand_addr.post_code, ts.vst_code as vehicle_slot_type_code,
+         case when ts.test_category_ref like 'ADI%' then cand_adi.prn else null end as candidate_prn,
+         org_reg.business_id as business_id,
+         org.organisation_name as business_name,
+         org.organisation_id as organisation_id,
+         org_reg.organisation_register_id as organisation_register_id,
+         bus_addr.address_id as business_addr_id,
+         bus_cd.contact_details_id as business_cd_id,
+         bus_addr.address_line_1 as business_addr_line1,
+         bus_addr.address_line_2 as business_addr_line2,
+         bus_addr.address_line_3 as business_addr_line3,
+         bus_addr.address_line_4 as business_addr_line4,
+         bus_addr.address_line_5 as business_addr_line5,
+         bus_addr.post_code as business_post_code,
+         bus_cd.primary_tel_number as business_telephone,
+         cancellations.cancel_initiator
+       from BOOKING  b
+         join APPLICATION a on a.app_id = b.app_id
+         join APPLICATION_RSIS_INFO ari on b.booking_id = ari.booking_id
+         left join VEHICLE v on a.vehicle_id = v.vehicle_id
+         left join TEST_SERVICE ts on a.test_service_id = ts.test_service_id
+         left join INDIVIDUAL  i on a.individual_id = i.individual_id
+         left join REF_DATA_ITEM_MASTER  title_ref on i.title_code = title_ref.item_id
+         left join CONTACT_DETAILS ccd on a.individual_id = ccd.individual_id
+         left join ADDRESS cand_addr on cand_addr.address_type_code = 1263 and a.individual_id = cand_addr.individual_id
+         left join REGISTER cand_adi on a.individual_id = cand_adi.individual_id
+         left join CUSTOMER_ORDER co on a.order_id = co.order_id and co.booker_type_code in ('B','T')
```

Comment:
Changing this to left join means that any applications without a corresponding customer order will be allowed through.  Is this an issue or even possible? (current MySQL data suggests not)